WordPress/src/main/res/drawable/ic_scan_idle_threats_found.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
⚠️ We usually name the resources `ic_[NAME]_[WIDTH]dp.xml`. I think it'd be good to update the names to keep the consistency. Thanks!